nomic/atlas.py

```diff
@@ -536,6 +540,7 @@ def add_text(self, project_id: str, data: List[Dict], shard_size=1000, num_worke
         * **data** - An [N,] element list of dictionaries containing metadata for each embedding.
         * **shard_size** - Embeddings are uploaded in parallel by many threads. Adjust the number of embeddings to upload by each worker.
         * **num_workers** - The number of worker threads to upload embeddings with.
+        * **replace_empty_values_with_null** - Replaces empty values in metadata with null. If false, will fail if empty values are supplied.
```

Comment:
documentations should specify that empty value is a zero length string, and the replacement will be a string with content null